lib/api/v3/activities/activity_representer.rb

```diff
@@ -29,7 +29,8 @@
 
 require 'roar/decorator'
 require 'roar/json/hal'
-require 'api/v3/utilities/date_time_formatter'
+
+API::V3::Utilities::DateTimeFormatter
```

Comment:
Note: As of now the `ActivityRepresenter` does not inherit from `Single` and thus we need to repeat ourselves here...